lib/active_campaign/v2/client.rb

```diff
@@ -66,7 +66,13 @@ def request(action, payload)
         response = self.class.send(action, *payload)
 
         if response.content_type == 'application/json'
-          JSON.parse(response)
+          response_json = JSON.parse(response)
```

Comment:
It would probably be smart to create a `parse_json` method that rescues the `JSON::ParserError` exception and returns a mock error response like `{"result_code":0,"result_message":"Invalid JSON in response"}`. I have seen too many APIs that generate invalid JSON sometimes.